example/notebooks/pneumonia.ipynb

```diff
@@ -0,0 +1,1083 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "# Explanation analysis using pneumonia XRay pictures with FoXAI\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Double check if you need to reinstall linux libaries related to OpenCV"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!apt-get update && apt-get install ffmpeg libsm6 libxext6  -y"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Installing most needed libaries "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip install scikit-learn foxai tqdm"
```

Comment:
I've got error with missing `ipywidgets` package in model training function call.

```suggestion
    "!pip install scikit-learn foxai tqdm ipywidgets"
```

Reply:
fixed